ru/mail/chrome/communicator/wallet/SignonViewer.dtd

```diff
@@ -1,14 +1,14 @@
-<!ENTITY      windowtitle.label               "Менеджер паролей">
+﻿<!ENTITY      windowtitle.label               "Менеджер паролей">
 <!ENTITY      windowalttitle.label            "Менеджер форм: Сайты">
-<!ENTITY      tab.signonsstored.label         "Сохранённые пароли">
-<!ENTITY      tab.signonsnotstored.label      "Пароли никогда не будут сохраняться">
-<!ENTITY      tab.nopreview.label             "Предварительно введённые в форму данные никогда не будут сохраняться">
-<!ENTITY      tab.nocapture.label             "Введённые в форму данные никогда не будут сохраняться">
+<!ENTITY      tab.signonsstored.label         "Сохраненные пароли">
```

Comment:
In practice, localizers do as they please lately. Some clients even stress that they would like to see "ё" letters on their web pages, since it adds a certain retro style. :) What I've really meant was consistency. So, if it is possible to replace ALL "ё" by "е" right in GitHub, let's do it. All in all, finalizing such texts is much more efficient when using CAT tools due to global find and replace features. If we switch to Trados or MemoQ + Apsic Xbench QA add-in, it could be great.